src/GeneralStateTestsFiller/stCreateTest/CreateAddressWarmAfterInitcodeFailureFiller.yml

```diff
@@ -0,0 +1,96 @@
+CreateAddressWarmAfterInitcodeFailure:
+  _info:
+    comment: |
+      Invokes failing CREATE (because initcode fails) and checks
+      if the create address is considered warm in the follow up call as required by EIP-2929.
+  env:
+    currentCoinbase: 2adc25665018aa1fe0e6bc666dac8fc2697ff9ba
+    currentDifficulty: 1
+    currentGasLimit: 30000000
+    currentNumber: 2
+    currentTimestamp: 999
+    previousHash: 5e20a0453cecd065ea59c37ac63e079ee08998b6045136a8ce6635c7912ec0b6
+
+  pre:
+    # Tx sender
+    a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+      nonce: 0
+      balance: 100000001
+      storage: {}
+      code: ''
+
+    00000000000000000000000000000000000c0deC:
+      nonce: 0
+      balance: 0
+      storage: {}
+      code: |
+        :yul
+          object "C" {
+            code {
+              let initcode_size := datasize("initcode")
+              datacopy(0, dataoffset("initcode"), initcode_size)
+              sstore(0, create(0, 0, initcode_size))
+              // Call the create address.
+              sstore(1, call(0, 0xd4e7ae083132925a4927c1f5816238ba17b82a65, callvalue(), 0, 0, 0, 0))
```

Comment:
Can you measure in expect section the gas price of call before and after it is warmed up.
By 2929. You can have smth like 
 sstore(10, gas())
Call()
Sstore(10, sub(sload(10), gas()))

Reply:
I added the code to check the costs of the `create` and `call`. But the effect of sending value is 34k gas, which is a lot more than the cost of cold vs. warm. See https://www.evm.codes/#f1 (or the comments I added to the filler).